src/Symfony/Component/Form/Tests/Extension/Core/Type/FormTypeTest.php

```diff
@@ -529,7 +529,7 @@ public function testOverrideErrorBubbling()
     public function testPropertyPath()
     {
         $form = $this->factory->create('form', null, array(
-            'property_path' => 'foo',
+            'map_to' => 'foo',
```

Comment:
You are right. I didn't consider that the old test methods are called `testPropertyPathXxx` while the new  ones would be called `testMapToXxx`.